openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -302,11 +302,24 @@ For more details and concrete mitigations, see (#security-considerations-pre-aut
 
 # Credential Offer Endpoint {#credential-offer-endpoint}
 
-This endpoint is used by a Credential Issuer that is already interacting with an End-User who wishes to initiate a Credential issuance. It is used to pass available information relevant for the Credential issuance to ensure a convenient and secure process.
+Credential Offer Endpoint is used by a Credential Issuer that is already interacting with an End-User who wishes to initiate a Credential issuance. It is used to pass available information relevant for the Credential issuance to ensure a convenient and secure process as defined in (#credential-offer).
+
+Credential Offer Endpoint can take one of the following values:
+
+- A case sensitive URL using the `https` scheme that contains scheme, host and, optionally, port number and path components, but no query or fragment components.
+- Custom URL scheme. For example, `openid-credential-offer://` as defined in (#client-metadata-retrieval).
+- Domain-bound Universal Links/App link.
+- No specific value. The issuer displays a QR code containing the Credential Offer, which the user can scan with a manually opened Wallet, or an arbitrary camera application on the user-device.
+
+How to obtain Credential Offer Endpoint is described in (#client-metadata-retrieval).
 
 ## Credential Offer {#credential-offer}
 
-The Credential Issuer sends Credential Offer using an HTTP GET request or an HTTP redirect to the Wallet's Credential Offer Endpoint defined in (#client-metadata).
+The Credential Issuer sends the Credential Offer using one of the following mechanisms:
+
+- Render a link that the End-User can click resulting in an HTTP GET request to the Credential Offer Endpoint.
```

Comment:
I don't think there is any reason to restrict this to a link. (A button would be fine.)

I think something like "Sends the user agent to the Credential Offer Endpoint" (i.e. similar language to how OAuth 2 talks about the redirect url) is sufficient, which covers all possible cases include a HTTP redirect. (It'd be fine to include "For example, a link ... or a HTTP redirect...".)

Reply:
I agree that this has to changed. But opening a native app is technically not sending the user agent to the credential offer endpoint. It is also not following a redirect nor a HTTP GET request. 

What about:

```suggestion
- Render a link that the End-User can click resulting in following the link to the Credential Offer Endpoint.
```